services/brig/src/Brig/Run.hs

```diff
@@ -103,3 +124,51 @@ lookupRequestIdMiddleware :: (RequestId -> Wai.Application) -> Wai.Application
 lookupRequestIdMiddleware mkapp req cont = do
   let reqid = maybe def RequestId $ lookupRequestId req
   mkapp reqid req cont
+
+cleanExpiredPendingInvitations :: AppIO ()
+cleanExpiredPendingInvitations = do
+  safeForever "cleanExpiredPendingInvitations" $ do
+    today <- utctDay <$> (liftIO =<< view currentTime)
```

Comment:
Why not `getCurrentTime`?  If this is in the `ReaderT`, does it ever change after boot time?

Reply:
I saw this pattern `liftIO =<< view currentTime` all over brig and assumed it was a convention.
The field is `_currentTime :: IO UTCTime`. I guessed it exists to mock a different time

Reply:
Sorry, you're right.  Didn't think that through.